internal/pkg/deploy/cloudformation/stack/name.go

```diff
@@ -3,7 +3,21 @@
 
 package stack
 
-import "fmt"
+import (
+	"fmt"
+	"strings"
+)
+
+// TaskStackPrefix is used elsewhere to list CF stacks
+const TaskStackPrefix = "task-"
```

Comment:
Do these entities need to be public? I couldn't find them being used by a client package